BO4E/COM/Bankverbindung.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.Json.Serialization;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.COM;
+
+/// <summary>Enthält eine Bankverbindung</summary>
+[ProtoContract]
+public class Bankverbindung : COM
+{
+    /// <summary>Die kontonummer. IBAN</summary>
+    [DataCategory(DataCategory.FINANCE)]
+    [JsonProperty(PropertyName = "kontonummer", Required = Required.Default, Order = 3)]
+    [JsonPropertyName("kontonummer")]
+    [ProtoMember(3)]
+    [JsonPropertyOrder(3)]
+    public string? Kontonummer { get; set; }
+
+    /// <summary>Der kontoinhaber</summary>
+    [DataCategory(DataCategory.FINANCE)]
+    [JsonProperty(PropertyName = "kontoinhaber", Required = Required.Default, Order = 4)]
+    [JsonPropertyName("kontoinhaber")]
+    [ProtoMember(4)]
+    [JsonPropertyOrder(4)]
+    public string? Kontoinhaber { get; set; }
+
+    /// <summary>Die Bankkennung, BIC</summary>
+    [DataCategory(DataCategory.FINANCE)]
+    [JsonProperty(PropertyName = "bankkennung", Required = Required.Default, Order = 5)]
+    [JsonPropertyName("bankkennung")]
+    [ProtoMember(5)]
+    [JsonPropertyOrder(5)]
+    public string? Bankkennung { get; set; }
```

Comment:
Was ist der Unterscheid zwischen Bankkennung und Bankcode? Bzw. Bankkennung ist ja die BIC, aber was ist dann der Code

Reply:
gute frage aber in template steht:
![image](https://github.com/user-attachments/assets/ac9d0c4e-16ed-4160-ab4d-04083b0dc51f)

Reply:
scheinbar das Bankcode ist niemals benutzt wurde, zumindest bei alle unsere examples

Reply:
und auch kein Land in EDIFACTS. können wir es weg lassen.

Reply:
https://github.com/Hochfrequenz/BO4E-dotnet/pull/466/commits/4727a191a47b45ed53cf9cdbe49f1386c8c498bd